src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -571,8 +560,7 @@ public function handleException($exception, array $error = null)
                 }
             }
         }
-        $exceptionHandler = $this->exceptionHandler;
-        $this->exceptionHandler = null;
+        $exceptionHandler = $this->setExceptionHandler();
```

Comment:
unrelated, I'd suggest reverting as the previous code required less back and forth to read